src/components/tools/text-toolbar.sass

```diff
@@ -0,0 +1,49 @@
+@import ../vars
+
+.text-toolbar
```

Comment:
`text-style-bar.*` => `text-toolbar.*`

The rendering is essentially unchanged aside from converting to a functional component and using the `useFloatingToolbarLocation()` hook. The rendering code will likely change substantially when the new design is implemented.